examples/gitlab/README.md

```diff
@@ -0,0 +1,46 @@
+## GitLab Push EventListener
+
+Creates an EventListener that listens for Gitlab webhook events. To expose the
+EventListener, add a `serviceType: LoadBalancer`.
```

Comment:
I'm a little confused by the `serviceType` line because there's a step below about port-forwarding. Both are options for exposing the EventListener pod, so I think we should probably mention them in the same section. 

Maybe mention the `serviceType: LoadBalancer` as a side-note if someone doesn't want to port-forward?

Reply:
added a note in the port forward section